content/en/api/dashboards/dashboards_create.md

```diff
@@ -38,5 +38,16 @@ external_redirect: /api/#create-a-dashboard
         The default value for the template variable on dashboard load.
     * **`prefix`** [*optional*, *default*=**None**]:
         The tag prefix associated with the variable. Only tags with this prefix appear in the variable dropdown.
+* **`template_variable_presets`** [*optional*, *default*=**None**]:
+    Array of Saved views for this Dashboards. Saved views definitions follow this form:
```

Comment:
I feel that `template variables saved views` is better than `Saved views` here, since this field is only tightly related with template variables and we didn't really define what is a dashboard saved view and which would be different from what the current `template_variable_presets`